src/DataTypes/Serializations/SerializationMap.cpp

```diff
@@ -98,17 +103,9 @@ void SerializationMap::serializeTextImpl(const IColumn & column, size_t row_num,
         if (i != offset)
             writeChar(',', ostr);
 
-        if (quote_key)
-        {
-            writeChar('"', ostr);
-            writer(key, nested_tuple.getColumn(0), i);
-            writeChar('"', ostr);
-        }
-        else
-            writer(key, nested_tuple.getColumn(0), i);
-
+        key_writer(key, nested_tuple.getColumn(0), i);
```

Comment:
`serializeTextImpl` have `ostr` parameter and it's captured by `KeyWriter` and `ValueWriter`. Does it always the same? If it is, maybe better to pass it explicitly here to writer?

Reply:
Yes, I agree that it's better to pass it explicitly.